examples/jitsi/jitsi-server.yaml

```diff
@@ -0,0 +1,476 @@
+  labels:
+    app.kubernetes.io/name: prosody
+    app.kubernetes.io/instance: jitsi
+type: Opaque
+data:
+---
+
+# ClusterIssuer
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: test@l7mp.io
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+    - http01:
+        ingress:
+          name: jitsi-meet
+---
+
+# Ingress
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: jitsi-meet
+  labels:
+    app.kubernetes.io/name: jitsi-meet
+    app.kubernetes.io/instance: jitsi
+    app.kubernetes.io/component: web
+  annotations:
+    kubernetes.io/ingress.class:  "nginx"
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+spec:
+  tls:
+    - secretName: jitsi-secret
+      hosts:
+      - <public-ingress-ip>.nip.io # TODO: Change it
+  rules:
+    - host: <public-ingress-ip>.nip.io # TODO: Change it
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: jitsi-meet
+                port:
+                  number: 80
+---
+
+# Jitsi web
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jitsi-web
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: jitsi-meet
+      app.kubernetes.io/instance: jitsi
+      app.kubernetes.io/component: web
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: jitsi-meet
+        app.kubernetes.io/instance: jitsi
+        app.kubernetes.io/component: web
+    spec:
+      containers:
+        - name: jitsi-meet
+          image: "jitsi/web:stable-7882"
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: config-volume
+              mountPath: /config/custom-config.js
+              subPath: custom-config.js
+          envFrom:
+          - configMapRef:
+              name: jitsi-meet
+          - configMapRef:
+              name: prosody-common
+          ports:
+            - name: http
+              containerPort: 80
+              protocol: TCP
+            - name: https
+              containerPort: 443
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: 80
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 80
+      volumes:
+        - name: config-volume
+          configMap:
+            name: jitsi-meet-config
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: jitsi-meet
+  labels:
+    app.kubernetes.io/name: jitsi-meet
+    app.kubernetes.io/instance: jitsi
+    app.kubernetes.io/component: web
+spec:
+  type: ClusterIP
+  ports:
+    - port: 80
+      protocol: TCP
+      name: http
+  selector:
+    app.kubernetes.io/name: jitsi-meet
+    app.kubernetes.io/instance: jitsi
+    app.kubernetes.io/component: web
+---
+
+# Prosody
```

Comment:
I'm wondering why Prosody needs a StatefulSet. Not that this was a problem, just wondering

Reply:
It's a StatefulSet because Prosody pods are not interchangeable.